nativelib/src/main/scala/scala/scalanative/posix/mqueue.scala

```diff
@@ -0,0 +1,48 @@
+package scala.scalanative
+package posix
+
+import native._
+import native.signal.sigevent
+import native.time.timespec
+import sys.types.{ssize_t, mode_t}
+
+// http://man7.org/linux/man-pages/man7/mq_overview.7.html
+
+@extern
+object mqueue {
+
+  def mq_open(name: CString,
+              oflag: CInt,
+              mode: mode_t,
+              attr: Ptr[mq_attr]): mqd_t                 = extern
```

Comment:
This is a binding for linux version, POSIX one differs: http://pubs.opengroup.org/onlinepubs/9699919799/functions/mq_open.html